issuance/issuer.go

```diff
@@ -152,7 +152,18 @@ func LoadChain(certFiles []string) ([]*Certificate, error) {
 
 // IssuerConfig describes the constraints on and URLs used by a single issuer.
 type IssuerConfig struct {
-	UseForRSALeaves   bool
+	// Active determines if the issuer can be used to sign precertificates. All
+	// issuers, regardless of this field, can be used to sign final certificates
+	// (for which an issuance token is presented), OCSP responses, and CRLs.
+	Active bool
+
+	// Deprecated: use Active instead. All Active issuers with RSA keys can be
+	// used for RSA leaves. No issuers with ECDSA keys can be used for RSA leaves.
+	UseForRSALeaves bool
+	// Deprecated: use Active instead. If features.ECDSAForAll is false, all
+	// Active issuers with RSA or ECDSA keys can be used for ECDSA leaves (and
+	// which is used will depend on the ECDSA Allow List). If features.ECDSAForAll
+	// is true, all Active issuers with ECDSA keys can be used for ECDSA leaves.
```

Comment:
This comment is describing what happens for the Active+ECDSAForAll combinations, but it's a comment on UseForECDSALeaves. Shouldn't it be discussing the UseForECDSALeaves+ECDSAForAll combinations?

Reply:
(See top-level comment about being synonyms for "active".) Discussing Active+ECDSAForAll and discussing UseForECDSALeaves+ECDSAForAll are the same thing.